plugins/out_s3/s3.c

```diff
@@ -105,7 +105,7 @@ static char *mock_error_response(char *error_env_var)
 
     err_val = getenv(error_env_var);
     if (err_val != NULL && strlen(err_val) > 0) {
-        error = flb_malloc(strlen(err_val) + sizeof(char));
+        error = flb_calloc(strlen(err_val) + sizeof(char));
```

Comment:
`flb_calloc` requires 2 paramters : element count and element size

In this case I'd suggest writing `flb_calloc(strlen(err_val) + 1, sizeof(char))`

Reply:
Fixed/Thanks!!!!

It is concerning that I
1) have known how calloc works in the past
2) self-code-reviewed this code to avoid embarrassing mistakes
3) didn't catch it

:/